ibis/pandas/tests/test_core.py

```diff
@@ -173,7 +173,13 @@ def is_computable_input_my_object(_):
     result = ibis.pandas.execute(four)
     assert result == 4.0
 
+    # Since execute_node is two level dispatcher, we need to delete it
+    # in multiple places
     del execute_node.funcs[ops.Add, int, MyObject]
+    del execute_node._meta_dispatcher.funcs[(ops.Add,)].funcs[
```

Comment:
may want to consider a method for this on the dispatcher itself? (other wise this leaks into the impl).

Reply:
Added __delitem__ for this